backend/config/bundles.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
```

Comment:
we need to create new bundles : 
1- common => the common code should be here, abstractions like(base controller)
2- painting  => the painting code should be here
3- core        => generic kafka, elastic search should be here